### PR TITLE
feat: replace cards with new actionCard component in manufacturing screens

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#81859

### CODE REVIEW

File: changelogs/unreleased/81859.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "title": "manufacturing screens: replace cards with new actionCard component",
```
Comment (vhu-axelor) [inline changelogs/unreleased/81859.json:2]: "Manufacturing screens: replace cards with actions by ActionCard"

File: packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx [hunk 2]
```diff
@@ -85,44 +88,40 @@ const ConsumedProductGlobalCard = ({
 
   return (
     <View style={styles.globalContainer}>
```
Comment (vhu-axelor) [inline packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx:90]: You can remove it

File: packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx [hunk 3]
```diff
@@ -85,44 +88,40 @@ const ConsumedProductGlobalCard = ({
...
+          },
+          {
+            iconName: 'plus-lg',
+            iconColor: Colors.primaryColor.background,
+            onPress: handleIncrement,
+            hidden: disableMore,
+          },
+        ]}>
+        <ConsumedProductCard
+          style={styles.consumedCard}
```
Comment (vhu-axelor) [inline packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx:113]: You can remove it

File: packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx [hunk 4]
```diff
@@ -85,44 +88,40 @@ const ConsumedProductGlobalCard = ({
...
-          style={styles.flexOne}
-          iconName="geo-alt-fill"
-          iconColor={Colors.secondaryColor.background}
-          onPress={onLocationPress}
+      <ActionCard
+        translator={I18n.t}
+        actionList={[
+          {
+            iconName: 'geo-alt-fill',
+            iconColor: Colors.secondaryColor.background,
```
Comment (vhu-axelor) [inline packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx:96]: You can remove it

File: packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx [hunk 5]
```diff
@@ -85,44 +88,40 @@ const ConsumedProductGlobalCard = ({
...
+            onPress: onLocationPress,
+          },
+          {
+            iconName: 'diagram-3-fill',
+            iconColor: Colors.secondaryColor.background,
```
Comment (vhu-axelor) [inline packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx:101]: You can remove it too

File: packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx [hunk 6]
```diff
@@ -84,9 +86,28 @@ const ConsumedProductGlobalCard = ({
...
+          onPress: onSubOfPress,
+          hidden: !isSubOF,
+        },
+        {
+          iconName: 'plus-lg',
+          onPress: handleIncrement,
+          hidden: disableMore,
+          iconColor: Colors.primaryColor.background,
+          large: true,
+        },
```
Comment (lme-axelor) [inline packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx:108]: Please add helper on actions : 
* "See availability" / "Voir la disponibilité"
* "See sub MO" / "Voir les sous OF"
* "Increment quantity" / "Augmenter la quantité"

**Pull request merged**